src/wallets/BalanceDisplay.scss

```diff
@@ -0,0 +1,24 @@
+@import '../themify', '../vars', '../partial';
+
+.BalanceDisplay {
+  margin: 0;
+
+  .balances {
+    .label {
+      font-size: $font-size-m;
+      font-weight: 600;
+    }
+
+    .available-balance {
+      @extend %monospace;
```

Comment:
Can't we use this "tabular figures" approach? https://github.com/JulietaUla/Montserrat/issues/101#issuecomment-579706104

Reply:
Good find! Sadly it still jumps around with this approach, so I'll stick with real monospace.